mypy/semanal.py

```diff
@@ -545,7 +545,57 @@ def check_function_signature(self, fdef: FuncItem) -> None:
         elif len(sig.arg_types) > len(fdef.arguments):
             self.fail('Type signature has too many arguments', fdef, blocker=True)
 
+    def check_namedtuple_classdef(self, defn: ClassDef) -> Tuple[List[str], List[Type]]:
```

Comment:
@elazarg 1600 is probably too far. I put it in the same place where other ``ClassDef`` helpers are (~ 750).